### DISCUSSION TITLE
Is Folding Static Code Blocks possible?

### DISCUSSION THREAD

**Discussion by Doshin108** [quarto-dev/quarto-cli#5329] [Q&A] [answered]
I'm using quarto for my documentation and I have static code blocks (sql) in my documentation.  Ideally I would like to fold those similarly to how code from executed code blocks can be done via option 'code-fold:true'

I have not had any success 'folding' static code block that are not executed. 
Does anyone have any insight?

**Comment by mcanouil** [comment]
See <https://github.com/quarto-dev/quarto-cli/discussions/5332#discussioncomment-5745023>.

**Reply by mcanouil** [reply]
To note, the related opened issue to track this #341

**Reply by Doshin108** [reply]
Thanks bud!  I'm watching it and the 3.4 dev stuff.      A redditer gave me a nice fix. :)

**Comment by moonpiedumplings** [comment, marked as answer]
You can use the html details tag to hide stuff, when writing in markdown.

Like so: 

```html
<details> 

<summary>summary here</summary>

Whatever you want here. Code, diagrams, images, you name it.

</details> 
```

Which renders to (I didn't know github comments could render html, pretty cool):

<details> 

<summary>summary here</summary>

Whatever you want here. Code, diagrams, images, you name it.

</details>

**Reply by Doshin108** [reply]
This is a really nice solution... Thank you.
